static/app/views/projectDetail/charts/projectBaseEventsChart.tsx

```diff
@@ -105,7 +105,6 @@ class ProjectBaseEventsChart extends Component<Props> {
             grid: {left: '10px', right: '10px', top: '40px', bottom: '0px'},
             yAxis: {
               axisLabel: {
-                color: theme.gray200,
```

Comment:
The default axis label color is `gray300`, so deleting this property would revert the color to that value.